meta-openpli/recipes-multimedia/dvd+rw-tools/files/fix-glibc2.28_major.patch

```diff
@@ -4,9 +4,9 @@
  #include <string.h>
  #include <fcntl.h>
  #include <sys/types.h>
-+#ifdef __GLIBC__
```

Comment:
here as well the question is: which libc could build without #include <sys/sysmacros.h>
This is needed both by glibc and musl . I'd remove the commented guards alltogether.